feature/src/main/resources/feature.xml

```diff
@@ -419,6 +547,25 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.labs/cloudsigma2-zrh/${jclouds.version}</bundle>
     </feature>
 
+    <feature name='jclouds-google-cloud-storage' description='Google Cloud Storage' version='${project.version}' resolver='(obr)'>
+        <feature version='${project.version}'>jclouds-blobstore</feature>
+        <bundle dependency='true'>mvn:org.apache.jclouds.api/oauth/${jclouds.version}</bundle>
+        <bundle dependency='true'>mvn:org.apache.jclouds.common/googlecloud/${jclouds.version}</bundle>
+        <bundle>mvn:org.apache.jclouds.labs/google-cloud-storage/${jclouds.version}</bundle>
+    </feature>
+
+    <feature name='jclouds-packet' description='Packet' version='${project.version}' resolver='(obr)'>
+        <feature version='${project.version}'>jclouds-compute</feature>
+        <bundle>mvn:org.apache.jclouds.labs/packet/${jclouds.version}</bundle>
+    </feature>
+
+    <feature name='jclouds-vagrant' description='Components to access Vagrant' version='${project.version}' resolver='(obr)'>
+        <feature version='${project.version}'>jclouds-compute</feature>
+        <bundle dependency='true'>mvn:name.neykov/vagrant-java-bindings/${vagrant-java-bindings.version}</bundle>
+        <bundle>mvn:org.apache.jclouds.labs/vagrant/${jclouds.version}</bundle>
+    </feature>    
+
+    <!-- SERVICES -->
```

Comment:
actually, I don't know how to split `feature.xml` into 2 files `{feature,feature-labs}.xml`, do I need to create another maven module called `feature-labs` maybe?